yarn.lock

```diff
@@ -1263,7 +1263,7 @@ base@^0.11.1:
 
 "bcoin@git+https://github.com/Airbitz/bcoin.git#primitiveBuild":
   version "1.0.0-beta.15"
-  resolved "git+https://github.com/Airbitz/bcoin.git#e1e5d9836d07ddae03fb31838dbea2118a15d4c7"
+  resolved "git+https://github.com/Airbitz/bcoin.git#cf7b7d63a0ed2b0c468672b01cdbdc511272aec7"
```

Comment:
I would be more comfortable with increasing the max fee in Bcoin vs removing it completely. This would catch any programmatic errors that spend too large a fee